### PR TITLE
online to connected

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation
 Online to connected in Dashboard

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/common/lang.js [hunk 1]
```diff
@@ -190,7 +190,6 @@ const lang = {
   NUMBEROFDEVICES: 'Number of devices',
   OCCURRENCE: 'Occurrence',
   OFFLINE: 'Offline',
-  ONLINE: 'Online',
```
Comment (ppathan) [inline src/common/lang.js]: Is ONLINE never used any other place?

File: src/components/deviceMap/regionDetails.js [hunk 2]
```diff
@@ -43,64 +43,51 @@ class RegionDetails extends Component {
...
+            <svg className=" total triangle">
+              <polygon points="25,2.5 48.5,49.8 2.5,49.8" fill="#fc540a" />
+            </svg>
+            <div className="critical">{lang.CRITICAL}</div>
+          </Col>
+          <Col md={6} className=" total-alarms-container warnings-container">
+            <div className=" total-alarms total-warnings">
+              {regionDetailsProps.totalWarningsDeviceCount}
+            </div>
+            <svg className="triangle rectangle">
```
Comment (spryor) [inline src/components/deviceMap/regionDetails.js]: Why are these SVGs embedded directly into the markup?

Comment (spryor) [inline src/components/deviceMap/regionDetails.js]: Why are these SVGs embedded directly into the markup?

File: src/components/deviceMap/regionDetails.js [hunk 3]
```diff
@@ -43,64 +43,51 @@ class RegionDetails extends Component {
...
-            </Col>
-          </Row>
-        </div>
+        <h3>{selectedDeviceGroupName}</h3>
+        <Row className="alarm-warning-container">
+          <Col md={6} className="total-alarms-container">
+            <div className="total-alarms">
+              {regionDetailsProps.totalAlarmDeviceCount}
+            </div>
+            <svg className=" total triangle">
```
Comment (spryor) [inline src/components/deviceMap/regionDetails.js]: space at the beginning of the class name

File: src/components/deviceMap/regionDetails.js [hunk 4]
```diff
@@ -43,64 +43,51 @@ class RegionDetails extends Component {
...
+            <div className="critical">{lang.CRITICAL}</div>
+          </Col>
+          <Col md={6} className=" total-alarms-container warnings-container">
+            <div className=" total-alarms total-warnings">
+              {regionDetailsProps.totalWarningsDeviceCount}
+            </div>
+            <svg className="triangle rectangle">
+              <rect width="8" height="8" fill="#FDE870" />
+            </svg>
+            <div className=" total warnings">
```
Comment (spryor) [inline src/components/deviceMap/regionDetails.js]: space at the beginning of the class name

File: src/common/lang.js (hunk 1 above)
Comment (swetha571) [reply src/common/lang.js]: No. We are not using anywhere.

**Pull request merged**